compliance/collection/compliance/sensorReplyHandler.go

```diff
@@ -0,0 +1,45 @@
+package compliance
+
+import (
+	"context"
+	"time"
+
+	"github.com/stackrox/rox/generated/internalapi/sensor"
+	"github.com/stackrox/rox/pkg/logging"
+)
+
+// SensorReplyHandlerImpl handles ACK/NACK messages from Sensor
+type SensorReplyHandlerImpl struct {
+	log         *logging.Logger
+	nodeScanner NodeScanner
+}
+
+// NewSensorReplyHandlerImpl returns new SensorReplyHandler
+func NewSensorReplyHandlerImpl(log *logging.Logger, nodeScanner NodeScanner) *SensorReplyHandlerImpl {
+	return &SensorReplyHandlerImpl{
+		log:         log,
+		nodeScanner: nodeScanner,
+	}
+}
+
+// HandleACK handles ACK message from Sensor
+func (s *SensorReplyHandlerImpl) HandleACK(_ context.Context, _ sensor.ComplianceService_CommunicateClient) {
+	s.log.Debugf("Received ACK from Sensor.")
+}
+
+// HandleNACK handles NACK message from Sensor
+func (s *SensorReplyHandlerImpl) HandleNACK(ctx context.Context, client sensor.ComplianceService_CommunicateClient) {
+	s.log.Infof("Received NACK from Sensor, resending NodeInventory in 10 seconds.")
+	go func() {
+		time.Sleep(time.Second * 10)
```

Comment:
Didn't we add a comment here about changing this into a ticker? I'll mention what we already discussed here just to not lose the comment:

You can `select` on the ticker and on a stop signal here. This is in case local-compliance has to shutdown before the 10s timer is over.